torch/csrc/jit/tensorexpr/kernel.h

```diff
@@ -141,7 +141,7 @@ class TensorExprKernel {
   std::unordered_map<int64_t, Tensor> tensors_;
   std::unordered_map<int64_t, Var> scalars_;
   std::unique_ptr<CodeGen> codegen_;
-  KernelArena kernel_arena_;
+  KernelScope kernel_scope_;
```

Comment:
I am not sure the semantics is correct. You could have multiple builds running at the same time, and they should bind their own scope when they are running.

Reply:
You're right, I missed that part.